packages/mermaid/src/diagrams/pie/pie.spec.ts

```diff
@@ -0,0 +1,178 @@
+// @ts-ignore: JISON doesn't support types
+import { parser } from './parser/pie.jison';
+import { DEFAULT_PIE_DB, db } from './pieDb.js';
+import { setConfig } from '../../config.js';
+
+setConfig({
+  securityLevel: 'strict',
+});
+
+describe('pie chart', () => {
+  beforeEach(() => {
+    parser.yy = db;
+    db.clear();
+    db.reset();
```

Comment:
I think `reset` is more general, but if we renamed it to `resetConfig`, then it would make sense, but we should document it somewhere to inform users about it because they might think clearing is related to DB only.